aptos-move/vm-genesis/src/lib.rs

```diff
@@ -417,12 +417,12 @@ fn initialize(
     // Calculate the per-epoch rewards rate, represented as 2 separate ints (numerator and
     // denominator).
     let rewards_rate_denominator = 1_000_000_000;
-    // Fold division of the percentage by 100 into multiplication with the denominator
-    // to minimize rounding errors due to integer division.
-    let rewards_rate_numerator = genesis_config.rewards_apy_percentage
-        * (rewards_rate_denominator / 100)
-        * genesis_config.epoch_duration_secs
-        / NUM_SECONDS_PER_YEAR;
+    let _num_epochs_in_a_year = NUM_SECONDS_PER_YEAR / genesis_config.epoch_duration_secs;
+    // Multiplication before division to minimize rounding errors due to integer division.
+    /*let rewards_rate_numerator = (genesis_config.rewards_apy_percentage * rewards_rate_denominator
+        / 100)
+        / num_epochs_in_a_year;*/
+    let rewards_rate_numerator = 0; // todo: feature flag this for Movement Labs
```

Comment:
A less intrusive way to effect this is to set `rewards_apy_percentage` to 0 in the config and relax the check in the `validate_genesis_config` function. That may be cleaner, although I don't see the config parameter used as source of truth for anything else in aptos-core code.

Reply:
Yeah, I'm sure there's a way. We can put in a small ticket for it. This is just to keep things moving and make sure we're not too all over the place.